bite-size-notes/week-8-b-probability-review.md

```diff
@@ -32,13 +31,17 @@ To start our probability review chapter, we'll first define some important terms
 
 **Common Notation**: Usually we will write $P(x)$ as the probability, a measure of chance between 0 and 1, of the random variable $X$ being $x$. So if we have the set $X$ representing a coin toss where the outcome can be heads or tails, we would denote $P(heads) = P(tails) = 0.5$. Note that the probability measure for each possible outcome considering a RV needs to sum to 1: $\sum P(x) = 1$. Additionally, we can have a vector of RVs, such as $\boldsymbol{X} = [X_1, X_2,...,X_n]$, this naturally gives us a multi-dimensional probability function $P(\boldsymbol{x}) = P(x_1, x_2,...,x_n)$. We can similarly have multiple random variables giving rise to multivariate probability functions. Given two RVs $X$ and $Y$, we can define $P(x,y) ~ (X,Y)$ and combining the two to get a multivariate, multi-dimensional probability space, we get $[(X_1, X_2,...,X_n), (Y_1, Y_2,...,Y_n)] ~ P(x_1, x_2,...,x_n,y_1,y_2,...,y_m)$. Since these probability operators can start to get quite long, bold symbols are usually used to denote vectors be sure to take note of the dimensionality of the systems being analyzed.
 
+## General Review
+
 **Probability Distributions**: Probability distributions describe the likelihood of picking a specific random value based on a probability density function (PDF). Where $X ~ P(x)$, we can describe the expected value (EV) of a probability function as $E(g(x)) = \int_x g(x) P(x) dx$. Here, $g(x) denotes a general function of the variable $x$, and $P(x)$ is the probability measure or distribution function. Note that probability distributions can take on either continuous or discrete forms, so the discrete version of a PDF is called a probability mass function (PMF). In discrete form, the expected value function $E(g(x)) = \sum_x g(x)P(x)$. 
 
```

Comment:
I think you can use $X$ ~ $P(X)$, otherwsie, there would be some errors.  Can you make some changes to this part? Adding some explanations. About XP(x) 
For the expectation, this will be helpful:$\mathbb{E}$.

Reply:
Thank you for the latex command, I couldn't find it when I was writing this. The tilde doesn't work for some reason in math mode so you have to exit math mode then reenter it.